draft-ietf-acme-acme.md

```diff
@@ -779,10 +779,12 @@ website (optional, string):
 information about the ACME server.
 
 caaIdentities (optional, array of string):
-: Each string MUST be a lowercase hostname which the ACME server recognizes as
-referring to itself for the purposes of CAA record validation as defined in
-{{!RFC6844}}.  This allows clients to determine the correct issuer domain name to
-use when configuring CAA records.
+: The hostnames that the ACME server recognizes as referring to
+itself for the purposes of CAA record validation as defined in
+{{!RFC6844}}.  Each string MUST represent the same sequence of ASCII
+code points that the server will expect to see as the "Issuer Domain
+Name.  This allows clients to determine the correct issuer domain
```

Comment:
You lost a closing `"` in this. Did you want to avoid the reference to the specific "issue/issuewild property tag"? I thought it was useful but I'm not attached if you disagreed.

Reply:
Nope, just didn't copy/paste properly 🙄